### PR TITLE
(New Feature)Certificate SSL and TLS Custom certificate validation

### PR DESCRIPTION
**Pull request by praneethreddy-4**
We can add the custom certificate and validate it.

Thank you for your contribution.
Before submitting this PR, please make sure:
- [ ] PR description and commit message should describe the changes done in this PR
- [ ] Verify the PR is point to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [ ] Latest Code from master or specific release branch is merged to your branch
- [ ] No unwanted\commented\junk code is included
- [ ] No new warning upon build solution
- [ ] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [ ] Sanity Tests are successfully executed for Existing Functionality
- [ ] After creating pull request there should not be any conflicts
- [ ] Resolve codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [ ] Resolve code review comments

### CODE REVIEW

File: Ginger/Ginger/Actions/ActionEditPages/Communication/ActeMailEditPage.xaml.cs [hunk 1]
```diff
@@ -74,6 +81,61 @@ private void Bind()
...
+        private void xcbValidationRequired_unchecked(object sender, RoutedEventArgs e)
+        {
+            IsValidationRequired.IsChecked = false;
+            CertificateStackPanel.Visibility = Visibility.Collapsed;
+        }
+        private void BrowseButton_Certificate(object sender, RoutedEventArgs e)
+        {
+            System.Windows.Forms.OpenFileDialog dlg = new System.Windows.Forms.OpenFileDialog();
+
+            dlg.DefaultExt = "*.crt";
```
Comment (Maheshkale447) [inline Ginger/Ginger/Actions/ActionEditPages/Communication/ActeMailEditPage.xaml.cs:98]: will it accept '.CRT' extension?

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]
Please handle codacy comments

File: Ginger/Ginger/Actions/ActionEditPages/Communication/ActeMailEditPage.xaml.cs (hunk 1 above)
Comment (praneethreddy-4) [reply Ginger/Ginger/Actions/ActionEditPages/Communication/ActeMailEditPage.xaml.cs]: Will accepts all the certifcate types which are possible.

**Comment by Maheshkale447** [comment]
Please add unit test cases to validate certificates

**Pull request merged**